### PR TITLE
librime: 1.5.3 -> 1.7.2

### PR DESCRIPTION
**Pull request by Vonfry**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 125609` at 9eab1eca run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/6d689dd6440ae1d403571228299ab0c0">brise</a></li>
    <li><a href="https://gist.github.com/c57f7878ab38f856c5ac6e2c18ac51e5">fcitx-engines.rime</a></li>
    <li><a href="https://gist.github.com/f01cff9209f356e4a53528243df45bb6">fcitx5-rime</a></li>
    <li><a href="https://gist.github.com/776647eb51e60c6c871c23843e3991f2">ibus-engines.rime</a></li>
    <li><a href="https://gist.github.com/0e0ccaa2dbbb2399af86fb7ad700817e">librime</a></li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

**Comment by Vonfry** [comment]
Update: add git clone recursive

File: pkgs/development/libraries/librime/default.nix [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+  version = "1.7.2";
 
   src = fetchFromGitHub {
     owner = "rime";
-    repo = "librime";
+    repo = pname;
     rev = version;
-    sha256 = "0xskhdhk7dgpc71r39pfzxi5vrlzy90aqj1gzv8nnapq91p2awhv";
+    sha256 = "027bav6h6k32ysxbf3jkvw5vhi6fpg1qn53f2pzpfvd0nn1s9fdh";
+    fetchSubmodules = true;
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/librime/default.nix:13]: Why do we need this?

Comment (Vonfry) [reply pkgs/development/libraries/librime/default.nix:13]: Its submodules are for dependencies, which already are done by nix. It was my fault

**Pull request merged**